server_admin/topics/authentication/conditions.adoc

```diff
@@ -41,3 +41,46 @@ Expected value in the attribute.
 Negate output:::
 You can negate the output.
 In other words, the attribute should not be present.
+
+==== Explicitly deny/allow access in conditional flows
+
+It's possible to explicitly either allow or deny access to the resources in a conditional flow.
```

Comment:
I suggest instead.
"You can allow or deny  access to resources in a conditional flow.